src/game.cpp

```diff
@@ -4801,137 +4803,90 @@ faction *game::list_factions(std::string title)
     return cur_frac;
 }
 
-void game::list_missions()
+void init_missions_tab( tabbed_window &win, input_broadcaster &ib, std::string &&tab_name, std::vector<mission *> &&data, std::string &&if_empty, bool can_confirm )
 {
-    WINDOW *w_missions = newwin(FULL_SCREEN_HEIGHT, FULL_SCREEN_WIDTH,
-                                (TERMY > FULL_SCREEN_HEIGHT) ? (TERMY - FULL_SCREEN_HEIGHT) / 2 : 0,
-                                (TERMX > FULL_SCREEN_WIDTH) ? (TERMX - FULL_SCREEN_WIDTH) / 2 : 0);
-
-    int tab = 0;
-    size_t selection = 0;
-    input_context ctxt("MISSIONS");
-    ctxt.register_cardinal();
-    ctxt.register_action("CONFIRM");
-    ctxt.register_action("QUIT");
-    ctxt.register_action("HELP_KEYBINDINGS");
```

Comment:
The new mission window misses a few key bindings:
- "CONFIRM", which is used to leave the window *and* set the currently selected mission to be the active mission.
- "QUIT" to exit it (currently it can not be exited at all, you have to kill the game).
- "HELP_KEYBINDINGS" to display the key bindings and allows to change them.